Description: Removed date from log file name, it confused logrotate.
Author: Charlie Smotherman <cjsmo@cableone.net>  

Index: ampache-3.6~alpha1/lib/log.lib.php
===================================================================
--- ampache-3.6~alpha1.orig/lib/log.lib.php	2011-05-08 22:25:46.000000000 -0500
+++ ampache-3.6~alpha1/lib/log.lib.php	2011-05-08 22:26:00.000000000 -0500
@@ -41,7 +41,7 @@
 	$log_name	= $log_name ? $log_name : 'ampache';
 	$username	= $username ? $username : 'ampache';
 
-	$log_filename	= Config::get('log_path') . "/$log_name.$log_day.log";
+	$log_filename	= Config::get('log_path') . "/$log_name.log";
 	$log_line	= "$log_time [$username] ($event_name) -> $event_description \n";
 
 	// Do the deed
